src/components/SideNav.jsx

```diff
@@ -83,19 +84,27 @@ const StyledContainer = styled.div`
   height: 100%;
   padding: 1.5rem;
   background: ${primaryGrey};
+  @media ${mobilePortrait}{
+    width: 60%;
+  }
 
   .navlinks {
      display: flex;
      flex-direction: column;
    }
 
+   @media ${tabletPortrait}{
+     height: 250vh;
+   }
+   /* I initially had 100vh on line 100 but that causes the bottom of the side bar to stop abruptly during the scroll down the page so I have changed it for now to 250vh. I am looking for a better solution. We could fix the menu on the left but this will involve more adjustments to the styling of the main content. (Lisa) */
+
```

Comment:
We can take a look at it together and resolve before merge...

Reply:
This is now resolved. :D